src/components/Reports/ReportsNew.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Helmet } from 'react-helmet';
+
+const ReportsNew = () => {
+  return (
+    <>
+      <FormattedMessage id="reports_title">
+        {(reports_title) => (
+          <Helmet>
+            <title>{reports_title}</title>
+          </Helmet>
+        )}
+      </FormattedMessage>
+      <section className="dp-container" style={{ marginTop: '20px' }}>
```

Comment:
Remember that for the margins and paddings we have the atomic classes and they are 6 in 6, in this case it is (m-t-24) and in the case of using a gray header it is not necessary, since the header has its margin

Reply:
yes! sorry i forgot.